libindy_vdr/src/ledger/request_builder.rs

```diff
@@ -711,17 +693,17 @@ impl RequestBuilder {
     pub fn build_rich_schema_request(
         &self,
         identifier: &DidValue,
-        rs_schema: RichSchema,
+        rich_schema: RichSchema,
     ) -> VdrResult<PreparedRequest> {
-        let rs_schema: RichSchema = RichSchema::new(
-            rs_schema.id,
-            rs_schema.content,
-            rs_schema.rs_name,
-            rs_schema.rs_version,
-            rs_schema.rs_type,
-            rs_schema.ver,
+        let rich_schema: RichSchema = RichSchema::new(
```

Comment:
Should `Clone` trait be derived/implemented for RichSchema struct?

Reply:
i will delete RichSchema object recreation at all